packages/odyssey-react-mui/src/Autocomplete.tsx

```diff
@@ -122,6 +122,7 @@ const Autocomplete = <
 
   return (
     <MuiAutocomplete
+      aria-disabled={isDisabled}
       disabled={isDisabled}
```

Comment:
**NON-BLOCKING**

We should only have `disabled` here as `aria-disabled` shouldn't change anything; it's only for screen readers.

Let's get a ticket made to document fixing this in the future to avoid to strange contrast error that shows up.

Reply:
Added a comment in the code.